compiler/zig/compiled-with-zig.yml

```diff
@@ -0,0 +1,25 @@
+rule:
+  meta:
+    name: compiled with Zig
+    namespace: compiler/zig
+    author: jakub.jozwiak@mandiant.com
+    scope: file
+    references:
+      - https://ziglang.org
+      - https://github.com/ziglang/zig
+    examples:
+      - 74e0758e469ab87f38cda7925d696ea41a122364891860e1ae30fc9ff2f68e7d
+  features:
+    - and:
+      - function:
+        - or:
+          - api: ntdll.NtLockFile
+          - api: ntdll.NtCreateFile
+          - api: ntdll.NtClose
+          - api: ntdll.NtCreateKeyedEvent
+          - api: ntdll.NtWaitForKeyedEvent
```

Comment:
this looks windows specific. should we add `os: windows` or include this characteristic in the rule name?

Reply:
Good point! I updated the rule and removed Windows specific entries. Strings seem to be specific enough so we should be good without that part.